transport/peer/hostport/peer_test.go

```diff
@@ -0,0 +1,162 @@
+package hostport
+
+import (
+	"testing"
+
+	"go.uber.org/yarpc/transport"
+	"go.uber.org/yarpc/transport/transporttest"
+
+	"github.com/golang/mock/gomock"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestPeerIdenfier(t *testing.T) {
+	tests := []struct {
+		hostport           string
+		expectedIdentifier string
+	}{
+		{
+			"localhost:12345",
+			"localhost:12345",
+		},
+		{
+			"123.123.123.123:12345",
+			"123.123.123.123:12345",
+		},
+	}
+
+	for _, tt := range tests {
+		pi := NewPeerIdentifier(tt.hostport)
+
+		assert.Equal(t, tt.expectedIdentifier, pi.Identifier())
+	}
+}
+
+func TestPeer(t *testing.T) {
```

Comment:
This test doesn’t look well-suited for table tests. I suggest breaking this into smaller methods with test utility functions for the little bit they share in common assertions.